builder-images/hacbs-jdk11-builder/Dockerfile

```diff
@@ -28,6 +28,14 @@ RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y apr
     && microdnf clean all \
     && rpm -q apr-devel autoconf automake bc buildah bzip2-devel cmake diffutils file findutils gcc gcc-c++ git glibc-devel glibc-devel.i686 glibc-langpack-en glibc-static gzip libcurl-devel libgcc.i686 libstdc++-static libtool lsof make openssl-devel podman shadow-utils tar unzip wget which zlib-devel
 
+RUN microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y emacs-filesystem vim-filesystem
+# XXX: RHEL 8 does not have ninja-build, so grabbing from CentOS 8
+RUN rpm --import https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official
+RUN wget --no-verbose https://vault.centos.org/centos/8/PowerTools/x86_64/os/Packages/ninja-build-1.8.2-1.el8.x86_64.rpm \
```

Comment:
If we are just wget'ing something we should probably check the hash is as expected.

I do wonder a bit how far we can go with native packages, at some point it may be we just have to leave it to PNC for now.

Reply:
This makes the assumption that the GPG key is valid and checks the GPG signature to verify the package.

Reply:
Specifically this was required to build BoringSSL.